zilliqa/src/consensus.rs

```diff
@@ -434,13 +437,19 @@ impl Consensus {
         // If we haven't applied the transaction yet, do so. This ensures we don't execute the transaction twice if we
         // already executed it in the process of proposing this block.
         if !self.transactions.contains_key(hash.0)? {
-            let mut listener = TouchedAddressEventListener::default();
-            let result = evm_ds::evm::tracing::using(&mut listener, || {
-                self.state
-                    .apply_transaction(txn.clone(), self.config.eth_chain_id, current_block)
-            })?;
-            for address in listener.touched {
-                self.touched_address_index.merge(address.0, hash.0)?;
+            let result = self.state.apply_transaction(
```

Comment:
we no longer use the touchedAddr..Listener as the infos is now contained in the tx result.